components/FoodGroups/Draggable/characterimages.tsx

```diff
@@ -0,0 +1,19 @@
+import { GROUPS } from '../groups'
+
+let notion_food_dict = new Map<string, GROUPS>([
+  // OLD - REMOVE ONCE DATABASE IS MIGRATED (Note that the dash is different from -)

```

Comment:
Address any changes needed here, I don't think it's finalised...? 😅

Reply:
This one needs changes on the notion database, we're not sure if changing the food group string will break other stuff so we havent done this. 
We're proposing to change the strings to something more simpler (for example instead of "Vegetables – protective foods" it is just "vegetable")

Reply:
I like that idea. Won't break anything pretty sure since the neither of the character pages use the foodGroup prop 👍